WNPRC_Purchasing/test/src/org/labkey/test/tests/wnprc_purchasing/WNPRC_PurchasingTest.java

```diff
@@ -779,7 +779,7 @@ public void testEmailNotificationRejectWorkflow()
         goToModule("Dumbster");
         EmailRecordTable mailTable = new EmailRecordTable(this);
         String subject = "Purchase request # " + requestId + " status update";
-        checker().verifyEquals("Incorrect To for the emails sent after rejection", Arrays.asList(getCurrentUser(), ADMIN_USER),
```

Comment:
same problem here which one is the right version to move forward

Reply:
See above comment